subprojects/parseq-restli-client/src/test/java/com/linkedin/restli/client/TestParSeqRestClient.java

```diff
@@ -156,6 +156,23 @@ public void testBatchingGetRequests() {
     }
   }
 
+  @Test
+  public void testBatchingGetRequestsForDiffKeyType() {
+    try {
+      setInboundRequestContext(new InboundRequestContextBuilder()
+          .setName("withBatching")
+          .build());
+      Task<?> t1  = greetingGet(1L);
+      Task<?> t2 = greetingGetForDiffKeyType("1");
+      Task<?> task = Task.par(t1, t2);
+      runAndWait(getTestClassName() + ".testBatchingGetRequestsForDiffKeyType", task);
+      assertFalse(hasTask("greetings batch_get(reqs: 1, ids: 2)", task.getTrace()));
+      assertTrue(hasTask("withTimeout 9999ms src: *.*/greetings.GET", task.getTrace()));
```

Comment:
What is the point of this check? Indicate that there are a single greeting get task?

Reply:
Yes, since these two requests are not batched together, they will send out two separate requests. I'm not sure if it's necessary to do this check.

Reply:
maybe just check `greeting get` task, timeout task has no context here for readers.